core/hash_ring.cpp

```diff
@@ -20,9 +20,18 @@
 
 namespace husky {
 
-void HashRing::insert(int worker_id, int num_ranges) { worker_set_.insert(worker_id); }
+void HashRing::insert(int worker_id, int num_ranges) {
+    worker_set_.push_back(worker_id);
+}
 
-void HashRing::remove(int worker_id) { worker_set_.erase(worker_id); }
+void HashRing::remove(int worker_id) {
+    for(int i=0; i<worker_set_.size(); i++) {
+        if(worker_set_[i] == worker_id) {
+            worker_set_.erase(worker_set_.begin()+i);
+            break;
+        }
+    }
```

Comment:
Is this ok? `worker_set_.erase(std::remove(worker_set_.begin(), worker_set_.end(), worker_id), worker_set_.end());`